### ISSUE TITLE
Double-check Weidy visitors being excluded

### ISSUE THREAD

**Issue by richarddubay** [NewSpring/Sprints#143]

**Comment by richarddubay** [comment]
https://github.com/NewSpring/Rock-Plugins/issues/93

**Comment by dcs619** [comment]
We need to find a way to exclude visitors from checking in by themselves

**Comment by benwiley** [comment]
I pulled up my family on the family select.  I added Bobby Visitor as a 2 year old male.  I checked Cullen and Bobby in.  Next I searched for Visitor (last name of new visitor) and got no results though he was added to the database and visible inside regular Rock UI.  I then added Robert Visitor to the family through the Rock UI.  I then was able to checkin Bobby by searching for Visitor.  This is the intended behavior, unless I am reading the comments wrong.
